packages/shadydom/src/patch-events.js

```diff
@@ -572,8 +582,8 @@ const EventPatchesDescriptors = utils.getOwnPropertyDescriptors(EventPatches);
 const SHADY_PROTO = '__shady_patchedProto';
 const SHADY_SOURCE_PROTO = '__shady_sourceProto';
 
-function patchEvent(event) {
-  event['__target'] = event.target;
+function patchEvent(event, forcedTarget = undefined) {
+  event['__target'] = forcedTarget !== undefined ? forcedTarget : event.target;
```

Comment:
```suggestion
function patchEvent(event, target = event.target) {
  event['__target'] = target;
```
I need to double check that parameter defaults can refer to earlier parameters but this would be clearer if so.

Reply:
Added.